src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/migration/MigrationAccount.java

```diff
@@ -16,62 +16,55 @@
 
 package xyz.groundx.caver_ext_kas.kas.wallet.migration;
 
-import org.web3j.utils.Numeric;
-
-import java.math.BigInteger;
+import java.util.List;
 
 /**
  * Representing a MigrationAccount which is used migrate external Klaytn account to KAS Wallet.
  */
 public class MigrationAccount {
+    /**
+     * An address of the account to be migrated.
+     */
     private String address;
+
+    /**
+     * The nonce value of the account to be migrated in hexadecimal format.
+     */
     private String nonce = "0x";
+
+    /**
+     * A instance of MigrationAccountKey representing various type of private key.
+     */
     private MigrationAccountKey<?> migrationAccountKey = null;
 
-    public MigrationAccount(MigrationAccount.Builder builder) {
-        this(builder.address,
-                builder.nonce,
-                builder.migrationAccountKey
-        );
+    public MigrationAccount(String address, String singlePrivateKey) {
+        this(address, "0x", singlePrivateKey);
+    }
+
+    public MigrationAccount(String address, String[] multisigPrivateKeys) {
+        this(address, "0x", multisigPrivateKeys);
     }
 
-    public MigrationAccount(String address, String nonce, MigrationAccountKey<?> migrationAccountKey) {
+    public MigrationAccount(String address, List<String[]> roleBasedPrivateKeys) {
+        this(address, "0x", roleBasedPrivateKeys);
+    }
+
+    public MigrationAccount(String address, String nonce, String singlePrivateKey) {
         setAddress(address);
         setNonce(nonce);
-        setMigrationAccountKey(migrationAccountKey);
+        setMigrationAccountKey(new SinglePrivateKey(singlePrivateKey));
     }
 
-    /**
-     * Represents a MigrationAccount class builder.
-     */
-    public static class Builder {
-        private String address;
-        private String nonce = "0x";
-        private MigrationAccountKey<?> migrationAccountKey = null;
-
-        public MigrationAccount.Builder setAddress(String address) {
-            this.address = address;
-            return this;
-        }
-
-        public MigrationAccount.Builder setNonce(String nonce) {
-            this.nonce = nonce;
-            return this;
-        }
-
-        public MigrationAccount.Builder setNonce(BigInteger nonce) {
-            this.nonce = Numeric.toHexStringWithPrefix(nonce);
-            return this;
-        }
-
-        public MigrationAccount.Builder setMigrationAccountKey(MigrationAccountKey<?> migrationAccountKey) {
-            this.migrationAccountKey = migrationAccountKey;
-            return this;
-        }
-
-        public MigrationAccount build() {
-            return new MigrationAccount(this);
-        }
+    public MigrationAccount(String address, String nonce, String[] multisigPrivateKeys) {
```

Comment:
@sirano11 
Should we add constructors that accept a Nonce value as a `BigInteger`?
```java
public MigrationAccount(String address, BigInteger nonce, String singlePrivateKey) {
        // ...
}
```